pkg/host/manager.go

```diff
@@ -45,7 +48,11 @@ func NewHostManager(utilsInterface utils.CmdInterface) HostManagerInterface {
 	sv := service.New(utilsInterface)
 	u := udev.New(utilsInterface)
 	v := vdpa.New(k, netlinkLib)
-	sr := sriov.New(utilsInterface, k, n, u, v, netlinkLib, dpUtils)
+	ib, err := infiniband.New(netlinkLib, k, n)
```

Comment:
thinking loud here. do we want to load the config file only once? if the user change the file without rebooting the node?

in general if possible I prefer that the New for the interfaces doesn't return errors

Reply:
The config is supposed to be statically created during the node provisioning and to not change throughout its lifecycle.
In case there's an urgent need to reload the config, the user can restart the config-daemon pod.